scripts/fixJsonSchema.js

```diff
@@ -0,0 +1,16 @@
+// This script exists to replace the root `anyOf` with `oneOf` in our JSON Schema.
+// The Rust Crate `schemars` is not properly outputting `oneOf` so we'll manually edit it here
+
+const fs = require('fs');
+
+const schemaPath = './website/static/relay-config-schema.json';
+
+const fileContents = fs.readFileSync(schemaPath, 'utf-8');
+const parsedFileContents = JSON.parse(fileContents);
+
+parsedFileContents.oneOf = parsedFileContents.anyOf;
```

Comment:
There should be some nested `anyOf`s that need to be covered as well if I remember correctly.

Reply:
Call me crazy, but I'm pretty sure all of the `anyOf`s should be converted to `oneOf`s?

Reply:
Yea, I agree. I don’t think there’s any anyOf in there that can’t be replaced by a oneOf.